### PR TITLE
Change target to es5

### PR DESCRIPTION
**Pull request by soswow**

### CODE REVIEW

**Comment by srijs** [comment]
Can you go into more detail why this is needed/desired, and couldn't be handled using a bundler?

File: tsconfig.json [hunk 1]
```diff
@@ -1,11 +1,17 @@
...
     "module": "commonjs",
-    "target": "es6",
+    "target": "es5",
     "noImplicitAny": true,
     "sourceMap": true,
     "declaration": true,
-    "outDir": "out"
+    "outDir": "out",
+    "lib": [
+      "dom",
```
Comment (srijs) [inline tsconfig.json]: I'm wondering why `dom` needed here, since this library should be independent of any browser-specific logic?

**Comment by soswow** [comment]
I believe when you define target `es5` it is added to libs automatically. But because I needed Promise one, I have to add all of them manually. Without it I get this:

screen shot 2018-04-15 at 8 37 58 am

**Comment by srijs** [comment]
Ah, cool, got it!

**Comment by soswow** [comment]
Nice. Thx. Tell me when you release a new version ;-)

**Comment by srijs** [comment]
Just published as 0.1.2 🎉

**Pull request merged**